packages/node_modules/@webex/internal-plugin-ediscovery/src/index.js

```diff
@@ -436,6 +436,12 @@ registerInternalPlugin('ediscovery', EDiscovery, {
             return object;
           }
 
+          if (!spaceSummary.participants || !spaceSummary.participants.length) {
+            ctx.webex.logger.error(`Known error - detected space with no participant - unable to decrypt name for space ${spaceSummary.spaceId}`);
+
+            return object;
```

Comment:
Should this and line 436 be `Promise.resolve(object)` since the other returns in this function are Promises?

Reply:
Thanks Adam, had a few of these further up too

Reply:
Good catch @adamweeks